### PR TITLE
fix an unnecessary re-initialization of Serial

### PR DESCRIPTION
**Pull request by meganetaaan**
If the third argument of M5.begin() is `true` it runs serial initialization. But it's not necessary because it's done by Serial.begin() above.

### CODE REVIEW

File: liff-app/liff.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // User service UUID: Change this to your generated service UUID
-const USER_SERVICE_UUID         = '91E4E176-D0B9-464D-9FE4-52EE3E9F1552'; // LED, Button
```
Comment (teknuze) [inline liff-app/liff.js]: would you kindly please change this back to the default starter UUID

**Review by teknuze — CHANGES_REQUESTED** [review]
@meganetaaan you're absolutely right

File: liff-app/liff.js (hunk 1 above)
Comment (meganetaaan) [reply liff-app/liff.js]: I've reverted this change.

**Review by teknuze — APPROVED** [review]

**Comment by teknuze** [comment]
Thank you!

**Pull request merged**